memgpt/local_llm/koboldcpp/api.py

```diff
@@ -0,0 +1,59 @@
+import os
+from urllib.parse import urljoin
+import requests
+import tiktoken
+
+from .settings import SIMPLE
+from ..utils import load_grammar_file
+from ...constants import LLM_MAX_TOKENS
+
+HOST = os.getenv("OPENAI_API_BASE")
+HOST_TYPE = os.getenv("BACKEND_TYPE")  # default None == ChatCompletion
+KOBOLDCPP_API_SUFFIX = "/generate"
```

Comment:
The api endpoint is the same as webui  "/api/v1/generate"

Reply:
just fixed in: https://github.com/cpacker/MemGPT/pull/293/commits/1ba25908746ba47a1ad4051d5c05eb136f6d676f

also the result unpackaging was wrong and needed a patch

seems to be working for kobold now on my end!